### PR TITLE
Add Openstack documentation for subctl

### PR DESCRIPTION
**Pull request by aswinsuryan**
Add documentation for using subctl with RHOS.

Signed-off-by: Aswin Suryanarayanan <aswinsuryan@gmail.com>

### CODE REVIEW

File: src/content/operations/deployment/subctl/_index.en.md [hunk 1]
```diff
@@ -405,6 +405,27 @@ This command prepares an OpenShift installer-provisioned infrastructure (IPI) on
 | `--natt-port` `<int>`            | IPsec NAT Traversal port (default 4500)
 | `--vxlan-port` `<int>`           | Internal VXLAN port (default 4800)
 
+#### `prepare rhos`
+
+`subctl cloud prepare rhos [flags]`
+
+This command prepares an OpenShift installer-provisioned infrastructure (IPI) on Openstack cloud for Submariner installation.
```
Comment (dfarrell07) [inline src/content/operations/deployment/subctl/_index.en.md]:
```suggestion
This command prepares an OpenShift installer-provisioned infrastructure (IPI) on OpenStack cloud for Submariner installation.
```

Per https://en.wikipedia.org/wiki/OpenStack and elsewhere. 

I can add this to our linting rules.

https://github.com/submariner-io/submariner-website/blob/devel/.markdownlint.yml#L21

File: src/content/operations/deployment/subctl/_index.en.md [hunk 2]
```diff
@@ -405,6 +405,27 @@ This command prepares an OpenShift installer-provisioned infrastructure (IPI) on
...
+
+| Flag                             | Description
+|:---------------------------------|:--------------------------------------------------------------------------------------------------------------------------|
+| `--cloud-entry` `<string>`       | the cloud entry to use (default `openstack`)
```
Comment (dfarrell07) [inline src/content/operations/deployment/subctl/_index.en.md]:
```suggestion
| `--cloud-entry` `<string>`       | the cloud entry to use (default `openstack`)
```

I know this is copied from subctl, but...as someone reading this for the first time, I'm left wondering: what does "entry" mean in this context? Entry as in "an entry in a list"? As in "entry point"? Just the name of the cloud from a specific set of names? Maybe we can use a different word or way of describing it in the Description, something that helps clarify what it means more than just repeating the words in the flag. Non-blocker for the docs as they are just copying subctl I think.

File: src/content/operations/deployment/subctl/_index.en.md (hunk 1 above)
Comment (dfarrell07) [reply src/content/operations/deployment/subctl/_index.en.md]: I sent a PR to lint for this: https://github.com/submariner-io/submariner-website/pull/730

Comment (aswinsuryan) [reply src/content/operations/deployment/subctl/_index.en.md]: Done.

File: src/content/operations/deployment/subctl/_index.en.md (hunk 2 above)
Comment (aswinsuryan) [reply src/content/operations/deployment/subctl/_index.en.md]: Done.

File: src/content/operations/deployment/subctl/_index.en.md [hunk 3]
```diff
@@ -405,6 +405,27 @@ This command prepares an OpenShift installer-provisioned infrastructure (IPI) on
...
+
+This command prepares an OpenShift installer-provisioned infrastructure (IPI) on OpenStack cloud for Submariner installation.
+
+| Flag                             | Description
+|:---------------------------------|:--------------------------------------------------------------------------------------------------------------------------|
+| `--cloud-entry` `<string>`       | the specific cloud configuration to use from the clouds.yaml (default `openstack`)
+| `--dedicated-gateway`            | Whether a dedicated gateway node has to be deployed (default true)
+| `--gateway-instance` `<string>`  | Type of gateway instance machine (default `PnTAE.CPU_4_Memory_8192_Disk_50`)
+| `--gateways` `<int>`             | Number of dedicated gateways to deploy (default 1)
+| `--infra-id` `<string>`          | Openstack infra ID
```
Comment (dfarrell07) [inline src/content/operations/deployment/subctl/_index.en.md]:
```suggestion
| `--infra-id` `<string>`          | OpenStack infra ID
```

There are a bunch of other instances of "Openstack->OpenStack".

**Review by maayanf24 — COMMENTED** [review]
I think that the entire part of `cloud prepare` should be reorganized, something similar to `diagnose`, where all shared flags are only described once, and not per single cloud.
I'll add a new issue to do the rewrite work

File: src/content/operations/deployment/subctl/_index.en.md (hunk 3 above)
Comment (aswinsuryan) [reply src/content/operations/deployment/subctl/_index.en.md]: Done.

**Review by maayanf24 — APPROVED** [review]

**Review by dfarrell07 — APPROVED** [review]

**Pull request merged**